androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/common/BackButton.kt

```diff
@@ -0,0 +1,41 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.common
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import co.nimblehq.avishek.phong.kmmic.android.R
+import co.nimblehq.avishek.phong.kmmic.android.ui.theme.ApplicationTheme
+
+@Composable
+fun BackButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Image(
+        painter = painterResource(id = R.drawable.ic_arrow_left),
+        contentDescription = null,
+        colorFilter = ColorFilter.tint(White),
+        contentScale = ContentScale.FillWidth,
```

Comment:
Do we need these value? I tried to run without them and it is still the same. If we don't need it then we don't have to separate this into a file.

Reply:
Updated in https://github.com/nimblehq/avishek-phong-kmm-ic/pull/118/commits/690207160547acd356edae11b175d058bc1a990f